### PR TITLE
MSDos CheatSheet

### PR DESCRIPTION
**Pull request by ShamimSuf**
https://duck.co/ia/view/msdos_cheat_sheet

### CODE REVIEW

**Comment by daxtheduck** [comment]
@ShamimSuf Hey!
        
Thanks for taking the time to contribute! We really appreciate it.

We work closely with every contributor to make Instant Answers the best they can be, so we appreciate your patience as we look over your code. From here, the process usually goes like this:

1. Pull Request is reviewed by the DuckDuckGo staff and community
2. Staff and community will leave feedback with any necessary updates to the function or design.
3. Once you've made any necessary corrections, then your Instant Answer will be merged and deployed live on DuckDuckGo!

If you have any questions along the way, feel free to ask them here. Our staff and community are also available on Slack to answer any questions you may have. If you'd like to join us there please send an email to <quackslack@duckduckgo.com>.

>More Info: https://duck.co/duckduckhack/submission_and_review

Thanks!

**Comment by daxtheduck** [comment]
## [Msdos Cheat Sheet](https://duck.co/ia/view/msdos_cheat_sheet)

**Description**: Cheat Sheet for Windows Command Prompt

**Example Query**: [msdos cheatsheet](https://beta.duckduckgo.com/?q=msdos cheatsheet), [dos cheatsheet, cmd cheatsheet](https://beta.duckduckgo.com/?q=dos cheatsheet, cmd cheatsheet)

**Tab Name**: Answer

**Source**: http://simplyadvanced.net/blog/cheat-sheet-for-windows-command-prompt/

*These are the important fields from the IA page.  Please check these for errors or missing information and update the [IA page](https://duck.co/ia/view/msdos_cheat_sheet)*

---
*This is an automated message which will be updated as changes are made to the [IA page](https://duck.co/ia/view/msdos_cheat_sheet)*

File: share/goodie/cheat_sheets/json/msdos.json [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+    "id": "msdos_cheat_sheet",
+    "name": "Msdos",
+    "description": "Cheat Sheet for Windows Command Prompt",
+    "metadata": {
+        "sourceName": "SimplyAdvanced Blog",
+        "sourceUrl": "http://simplyadvanced.net/blog/cheat-sheet-for-windows-command-prompt/"
+    },
+    "aliases": [
+        "dos",
+        "cmd",
```
Comment (moollaza) [inline share/goodie/cheat_sheets/json/msdos.json]: Please remove `cmd`. It is already an alias for another cheat sheet: https://duckduckgo.com/?q=cmd+cheat+sheet&ia=answer

**Comment by daxtheduck** [comment]
Automated screenshot from [beta.duckduckgo.com](https://beta.duckduckgo.com/?q=msdos cheatsheet)
                
                [Mobile screenshot](https://ia-screenshots.s3.amazonaws.com/msdos_cheat_sheet_mobile.png)

**Comment by daxtheduck** [comment]
Automated screenshot from [beta.duckduckgo.com](https://beta.duckduckgo.com/?q=msdos cheatsheet)
                
                [Mobile screenshot](https://ia-screenshots.s3.amazonaws.com/msdos_cheat_sheet_mobile.png)

**Comment by tagawa** [comment]
@ShamimSuf, thanks again for creating the MS-DOS cheat sheet! You’ve been doing a really good job here and we'd like you to help make DuckDuckGo even better for yourself and others interested in MS-DOS.

When you're ready to move on, please:

- Create a new Instant Answer for searches you do related to MS-DOS

OR

- Test live MS-DOS Instant Answers (or those in development) for bugs and suggest improvements

Detailed instructions explaining how you can help are available at the top of this [page](https://github.com/duckduckgo/duckduckgo/wiki/Creating-or-Improving-Instant-Answers).

Thanks again!

File: share/goodie/cheat_sheets/json/msdos.json [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+        "microsoft dos"
+    ],
+    "template_type": "code",
+    "section_order": [
+        "Basic Command Prompt Commands"
```
Comment (moollaza) [inline share/goodie/cheat_sheets/json/msdos.json]: These need to be split into more than 1 section. Ideally logical groupings of commands

File: share/goodie/cheat_sheets/json/msdos.json [hunk 3]
```diff
@@ -0,0 +1,126 @@
...
+    "metadata": {
+        "sourceName": "SimplyAdvanced Blog",
+        "sourceUrl": "http://simplyadvanced.net/blog/cheat-sheet-for-windows-command-prompt/"
+    },
+    "aliases": [
+        "dos",
+        "cmd",
+        "microsoft dos"
+    ],
+    "template_type": "code",
```
Comment (moollaza) [inline share/goodie/cheat_sheets/json/msdos.json]: Please switch this to the `terminal` template -- it will work well for this content

**Comment by daxtheduck** [comment]
Automated screenshot from [beta.duckduckgo.com](https://beta.duckduckgo.com/?q=msdos cheatsheet)
                
                [Mobile screenshot](https://images.duckduckgo.com/iu/?u=https://ia-screenshots.s3.amazonaws.com/msdos_cheat_sheet_mobile.png)

**Comment by daxtheduck** [comment]
Automated screenshot from [beta.duckduckgo.com](https://beta.duckduckgo.com/?q=msdos cheatsheet)
                
                [Mobile screenshot](https://images.duckduckgo.com/iu/?u=https://ia-screenshots.s3.amazonaws.com/msdos_cheat_sheet_mobile.png)

File: share/goodie/cheat_sheets/json/msdos.json (hunk 1 above)
Comment (moollaza) [inline share/goodie/cheat_sheets/json/msdos.json]: @ShamimSuf once this is done we can merge this Pull Request -- right now it's causing the tests to fail 😦

**Comment by mintsoft** [comment]
@mentions I'm going to close this PR for now; feel free to reopen it whenever you like :+1:

**Pull request closed without merging**